src/main/java/uk/gov/hmcts/reform/idam/api/fr/shared/model/ApiError.java

```diff
@@ -146,7 +150,7 @@ public String toString() {
    * Convert the given object to string with each line indented by 4 spaces
    * (except the first line).
    */
-  private String toIndentedString(Object o) {
+  private String toIndentedString(java.lang.Object o) {
```

Comment:
did it generate with `java.lang.Object`?

Reply:
Yes, this comes from the generator.